code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -49,6 +49,14 @@
 	icon_state = "tile-wood"
 	turf_type = /turf/simulated/floor/wood
 
+/obj/item/stack/tile/wood/cyborg

```

Comment:
@ProperPants for redefining behaviour specifically for cyborgs
these are literally identical, they have no reason to exist until they change something